File: Task-03/Python/SubTask3.py
n=int(input("input a number"))
for i in range(n//2 + 1):
    print(' ' * (n//2 - i), end='')
    print('*' * (2 * i + 1))
for i in range(n//2 - 1, -1, -1):
    print(' ' * (n//2 - i), end='')
    print('*' * (2 * i + 1))

File: Task-03/JavaScript/Subtask-3.js
function generateDiamond(n) {
    for (let i = 1; i <= n; i += 2) {
        const spaces = " ".repeat((n - i) / 2);
        const stars = "*".repeat(i);
        console.log(spaces + stars + spaces);
    }

    for (let i = n - 2; i >= 1; i -= 2) {
        const spaces = " ".repeat((n - i) / 2);
        const stars = "*".repeat(i);
        console.log(spaces + stars + spaces);
    }
}

const userInput = prompt("Enter a number to generate a diamond pattern:");
const n = parseInt(userInput);

if (!isNaN(n) && n > 0) {
    generateDiamond(n);
} else {
    console.log("Invalid input. Please enter a positive number.");
}
